R/tooltip.R

```diff
@@ -0,0 +1,117 @@
+#' Add a tooltip to a UI element
+#'
+#' Tooltips are useful for showing additional information when focusing (or
+#' hovering over) a UI element.
+#'
+#' @param ... Unnamed arguments can be any valid child of an [htmltools
+#'   tag][htmltools::tags]. Named arguments become HTML attributes on returned
+#'   UI element. Attributes starting with `data-bs-*` may be supplied to further
+#'   customize [tooltip
+#'   options](https://getbootstrap.com/docs/5.3/components/tooltips/).
+#' @param body Content to display in the tooltip.
```

Comment:
It's a minor thing but I'm not feeling `body` as the argument name. What do you think about `content` or `tooltip`? Or `title` (as it's called in Bootstrap)?

```r
tooltip(
  tag = a(href = "#", "Some text"),
  tooltip = HTML("Tooltip <em>message</em>"),
  placement = "below"
)
```

Or if we use `...` for the tooltip content, we wouldn't need a specific name (which I think is a reasonable argument in favor of using the `...`).

Reply:
I could get on board with `content`. FWIW, the name `body` came from the fact that `body` & `title` makes sense for popovers (especially since it's kind of like a card), and I was hoping to land on a name that would give basically the same result (whether you're using `tooltip()` or `popover()`)

Reply:
Ah, that makes sense. I think `content` would be better then. But I do think I'd personally put this in the `...` argument. It definitely makes sense to me that `popover()` would receive `...` and place them in the body and that `title` is an argument taking a single value.